filebeat/config/config.go

```diff
@@ -86,6 +87,12 @@ type MultilineConfig struct {
 	Timeout  string `yaml:"timeout"`
 }
 
+type JsonDecoderConfig struct {
+	OnUnmarshalError string `yaml:"on_unmarshal_error"`
```

Comment:
And maybe even take the value and run `strings.ToLower()` on it so that it's case insensitive.